common/autoconf/jdk-options.m4

```diff
@@ -165,9 +165,6 @@ AC_DEFUN_ONCE([JDKOPT_SETUP_JVM_VARIANTS],
   if test "x$VAR_CPU" = xppc64 -o "x$VAR_CPU" = xppc64le ; then
     INCLUDE_SA=false
   fi
-  if test "x$OPENJDK_TARGET_CPU" = xaarch64; then
-    INCLUDE_SA=false
-  fi
```

Comment:
I don't think we want this change.

Reply:
This is from OpenJDK, is there an issue/pr that requires the code snippet being deleted?

Reply:
The change is to delete those lines. We don't want that; we want to continue to support aarch64, so those lines should stay.

Reply:
As per https://bugs.openjdk.java.net/browse/JDK-8155627, this change is to _Enable SA on AArch64_ 
        _The serviceability agent is currently disabled. There is no good reason for this._
I think OpenJ9 can use this change to enable SA.

Reply:
If we think SA should be enabled, it should be done in a separate pull request specifically for that purpose.

Reply:
This commit comes in with OpenJDK updates as part of merging process, if a change to this OpenJDK commit (i.e., JCL patch) is required, it should be in a separated PR instead, does this make sense?

Reply:
I misunderstood the source of those lines. They originated with openjdk and were not changed for openj9. We should follow openjdk and remove them too. Sorry for the diversion.